conans/client/tools/files.py

```diff
@@ -8,12 +8,21 @@
 
 from conans.client.output import ConanOutput
 from conans.errors import ConanException
-from conans.util.files import load, save, _generic_algorithm_sum
+from conans.util.files import (load, save, _generic_algorithm_sum,
+                               mkdir_tmp as _mkdir_tmp, tmp_file as _tmp_file)
 
 
 _global_output = None
 
 
+def mkdir_tmp():
```

Comment:
You don't need to re-implement them to expose them to consumer. Just the import is fine.

Reply:
I know. But the imports tend to disappear even with the ugly tag of the unused bla bla.

Reply:
Anyway, Two birds with one stone, I will remove them.